flower_classifier/datasets/oxford_flowers.py

```diff
@@ -0,0 +1,93 @@
+import logging
+
+import torch
+from pathlib import Path
+
+from torch.utils.data import Dataset
+import torchvision
+from PIL import Image
+from scipy.io import loadmat
```

Comment:
I just learned more about the Pep8 import standards and was just curious if this is more appropriate?
```
import logging
from pathlib import Path

from PIL import Image
import torch
from torch.utils.data import Dataset
import torchvision
from scipy.io import loadmat
```

Reply:
Good reminder! That's close, but we'll typically use a tool like `isort` which will automatically sort the imports and will put all of the `import` statements above the `from` statements in each section. 

```
import logging
from pathlib import Path

import torch
import torchvision
from PIL import Image
from scipy.io import loadmat
from torch.utils.data import Dataset
```

I have an issue created (#7) to set up an automatic check to make sure we're formatting the code and sorting the imports.